### DISCUSSION TITLE
<My account has been flagged with no update>

### DISCUSSION THREAD

**Discussion by librarydemise457892** [community/community#47820] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My account was wrongly flagged almost immediately upon creation. I wrote to support and did not hear back so I followed up with another email and still have not heard back. Does anyone know roughly how long this will take? That is if you can even see this post.

**Comment by MateuszRL96** [comment]
Unfortunately, support response times can vary depending on the volume of requests. In some cases, it may take a few days to a week or more. If you haven't heard back after 7 business days, I recommend checking your spam/junk folder in case the reply was filtered.

If there's still no response, you could try reaching out again or see if there's an alternative contact method—some platforms also monitor posts on forums or social media. Hopefully, support will get back to you soon and resolve the issue.

Good luck, and I hope it gets sorted out quickly!

**Comment by kenil-gopani** [comment]
### ✅ What You Can Do

1. **Submit a ticket again** through [[GitHub Support](https://support.github.com/contact)](https://support.github.com/contact), choosing "Account flagged" as the topic.

2. **Wait patiently** – multiple submissions won't speed things up and can potentially delay your ticket further due to duplicate tracking.

3. **Avoid public GitHub Discussions for account-specific issues** – other users can’t help with account flags. Only GitHub staff can resolve this.

---

### ⚠️ Note:

GitHub staff **do not monitor or respond** to support issues through Community Discussions. It's mostly user-to-user help. If you're locked out of your account and can't submit a support ticket from the affected account, try submitting from another email, clearly mentioning your original username/email and the issue.

---

PLEASE MARK AS A ANSWER
